### PR TITLE
various style changes to the app header

### PR DESCRIPTION
**Pull request by Johnr302**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
     monospace;
 }
+
+/* #btn {
+  border-radius: 8px;
+  height: 35px;
+  width: 114px;
+  font-size: 18px;
+  background-color: #ffffff;
+} */
```
Comment (d48) [inline src/index.css:21]: @Johnr302 remove commented out code if not being used

File: src/AppHeader.css [hunk 2]
```diff
@@ -1,12 +1,13 @@
 .App-header {
   background-color: #212121;
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  grid-template: "leftHeader rightHeader";
+  /* flex-direction: row; */
   align-items: center;
-  justify-content: left;
+  /* justify-content: left; */
```
Comment (d48) [inline src/AppHeader.css:7]: remove commented code if not being used. same comment throughout the files

File: src/AppHeader.css [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
   font-size: calc(10px + 2vmin);
-  color: white;
-  height: 64px;
+  color: FFF;
```
Comment (d48) [inline src/AppHeader.css:9]: should be `#fff`. let's always have css lowercase hex codes as well

File: src/index.css [hunk 4]
```diff
@@ -12,10 +12,10 @@ code {
     monospace;
 }
 
-/* #btn {
+#btn {
   border-radius: 8px;
   height: 35px;
   width: 114px;
   font-size: 18px;
   background-color: #ffffff;
```
Comment (d48) [inline src/index.css:20]: is this block being used?

File: src/AppHeader.css [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  grid-template: "leftHeader rightHeader";
   align-items: center;
-  justify-content: left;
   font-size: calc(10px + 2vmin);
-  color: white;
-  height: 64px;
+  color: fff;
```
Comment (d48) [inline src/AppHeader.css:7]: needs to have hashtag in front:

```css
color: #fff;
```

File: src/index.css (hunk 4 above)
Comment (d48) [reply src/index.css:20]: also color properties that are more that are the same character greater than 3 can be converted to shorthand like:

```css
background-color: #fff;
```
That's a general convention that is used.

Comment (Johnr302) [reply src/index.css:20]: It's to style buttons.

File: src/index.css [hunk 6]
```diff
@@ -1,13 +1,21 @@
...
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+#btn {
```
Comment (d48) [inline src/index.css:15]: from our talk it looks like this was supposed to be a class. If so, update and this is good to go

**Pull request merged**